test-clients/src/main/java/com/hedera/services/bdd/suites/reconnect/AutoRenewEntitiesForReconnect.java

```diff
@@ -104,5 +105,25 @@ private HapiApiSpec accountAutoRenewalSuiteCleanup() {
 	protected Logger getResultsLogger() {
 		return log;
 	}
+
+	/**
+	 * Since reconnect is not supported when node starts from genesis, run some transactions before running correctness
+	 * tests so that a state is saved before reconnect.
+	 *
+	 * @return
+	 */
+	public static HapiApiSpec runTransfersBeforeReconnect() {
+		return defaultHapiSpec("runTransfersBeforeReconnect")
+				.given().when().then(
+						// do some transfers to save a state before reconnect
+						withOpContext((spec, ctxLog) -> {
+							List<HapiSpecOperation> opsList = new ArrayList<HapiSpecOperation>();
+							for (int i = 0; i < 500; i++) {
+								opsList.add(cryptoTransfer(tinyBarsFromTo(GENESIS, NODE, 1L)).logged());
```

Comment:
I think `.logged()` could be removed. It seems to have been useful for your debug before only.

Reply:
same here . It was added for debug purpose. Will remove now. Thanks